scripts/mysql_install_db.sh

```diff
@@ -127,7 +130,7 @@ parse_arguments()
       --verbose) verbose=1 ;; # Obsolete
       --rpm) in_rpm=1 ;;
       --help) usage ;;
-      --no-defaults|--defaults-file=*|--defaults-extra-file=*)
+      --no-defaults|--defaults-file=*|--defaults-extra-file=*|--defaults-group-suffix=*)
```

Comment:
What are you aiming to improve with this? Multi-instance systemd service?

Shouldn't we concatenate new args with old ones? E.g. shouldn't this work `--defaults-file=svojdb.cnf --defaults-group-suffix=instance143`?

Reply:
The main goal was the systemd multi-instance serve. Not sure if having mysql_install_db there by default is going to be accepted however at least it can be there if users choice it.

Good point about concatenation. Done now.